.storybook/stories/button/button-group.stories.ts

```diff
@@ -56,9 +58,20 @@ const defaultParameters: Parameters = {
     content: 'Hello World!',
     buttonCount: 3,
     inMenuButtonCount: 3,
+    disabledButtonsPosition: [],
   },
 };
 
-const variants: Parameters[] = [];
+setupStorybook(ClrButtonGroupModule, defaultStory, defaultParameters, generateVariants());
 
-setupStorybook(ClrButtonGroupModule, defaultStory, defaultParameters, variants);
+function generateVariants() {
+  const variants: Parameters[] = [];
+
+  for (const disabledButtonsPosition of ['', '2', '2,3']) {
```

Comment:
This won't work now. I missed this in the "other changes" list in my array control comment. Sorry!

Also, I think this one is simple enough to just be an array. We currently only use a `generateVariants` function when there are nested loops.

```
const variants: Parameters[] = [
  {
    disabledButtonsPosition: [],
  },
  {
    disabledButtonsPosition: [2],
  },
  {
    disabledButtonsPosition: [2,3],
  },
];
```

Reply:
sure I will update the above logic also